slc_ci_setup/cli/commands/deploy_ci_build.py

```diff
@@ -0,0 +1,28 @@
+import click
+
+from slc_ci_setup.cli.cli import cli
+from slc_ci_setup.cli.common import add_options
+from slc_ci_setup.lib.run_deploy_ci_build import run_deploy_ci_build
+from slc_ci_setup.cli.options.aws_options import aws_options
+
+
+CODE_BUILD_STACK_NAME = "CIBuild"
+SLC_PROJECT = "ScriptLanguages"
+SLC_RELEASE_PROJECT = "ScriptLanguagesRelease"
+
+GITHUB_URLS = {
+    SLC_PROJECT : "https://github.com/exasol/script-languages-release",
+    SLC_RELEASE_PROJECT : "https://github.com/exasol/script-languages"
+}
+
+
+@cli.command()
+@add_options(aws_options)
+@click.option('--project', type=click.Choice([SLC_PROJECT,  SLC_RELEASE_PROJECT]), default="""ScriptLanguagesRelease""",
```

Comment:
If we split the flavors, we get more repos. So, using choice is not a good idea

Reply:
I extracted that part into another class: slc_projects.py. I believe it makes maintenance easier, but I think it's important to have a sanity check here.

Reply:
Sanity check I agree, Repo limit, I don't agree. The sanity check should consist of checking if the structure fits, the required tools are available, ...

Reply:
If you think, it's useful, you can also keep the repository name check and print a warning and confirmation prompt, if the repo is not in the list.

Reply:
Removed the hard coded list. Clients need to add the Project URL now as parameter.